IntelFsp2Pkg/FspSecCore/SecMain.c

```diff
@@ -177,7 +177,7 @@ SecStartup (
   //
   // Call PeiCore Entry
   //
-  PeiCore (&SecCoreData, mPeiSecPlatformInformationPpi);
+  PeiCore (&SecCoreData, mPeiSecPlatformInformationPpi, 0);

```

Comment:
I think this call points to the wrapper function "ModuleEntryPoint" defined in MdePkg\Library\PeiCoreEntryPoint\PeiCoreEntryPoint.c.   
VOID
EFIAPI
_ModuleEntryPoint(
  IN CONST  EFI_SEC_PEI_HAND_OFF    *SecCoreData,
  IN CONST  EFI_PEI_PPI_DESCRIPTOR  *PpiList
  );

And this wrapper function will eventually call into PciCore() in PeiMain.c you pointed out. 
The PEI core entry is patched by build script:
https://github.com/universalpayload/fspsdk/blob/9d7fee5f01c5438a077d29cffcb81aba639f3a7f/BuildFsp.py#L397

I think it is 2 parameter.

Reply:
Thank you for your insight. I studied the links you provided and believe you are right.

The only difference I see between these two declarations is the "EFIAPI". I will test to see whether this makes a difference and re-upload my change for review.

Reply:
Thank you, I updated my commit.